src/core/env/prod/terraform.tfvars

```diff
@@ -30,6 +30,7 @@ cidr_subnet_fnelt                          = ["10.0.11.0/24"]
 cidr_subnet_fnpblevtdispatcher             = ["10.0.12.0/24"]
 cidr_subnet_appgateway                     = ["10.0.13.0/24"]
 cidr_subnet_redis_apim                     = ["10.0.14.0/24"]
+cidr_subnet_shared_1                       = ["10.0.16.0/26"]
```

Comment:
NOTE: .15 is being proposed for `functions-admin` in https://github.com/pagopa/io-infra/pull/315